lib/curation_concerns/engine.rb

```diff
@@ -18,6 +18,7 @@ class Engine < ::Rails::Engine
     require 'breadcrumbs_on_rails'
 
     config.autoload_paths += %W(
+      #{config.root}/app/actors/concerns
```

Comment:
This was directly from models. I'm :+1: on changing it, but not as part of this commit.